wwwapp/admin.py

```diff
@@ -11,6 +12,16 @@
 
 class UserProfileInline(admin.StackedInline):
     model = UserProfile
+    readonly_fields = ['userinfo_link', ]
+
+    def userinfo_link(self, instance):
+        # TODO: Render it as an inline instead - I just can't get it to work with a nested relation like that...
```

Comment:
Can anybody get this to work? I got tired of trying :stuck_out_tongue:

Reply:
I got have something like this working in other app. I will see how I did it.

Reply:
I misunderstood the problem. I don't know how to do this :(